pulsar/internal/topic_name_test.go

```diff
@@ -43,6 +43,12 @@ func TestParseTopicName(t *testing.T) {
 	assert.Equal(t, "my-tenant/my-namespace", topic.Namespace)
 	assert.Equal(t, -1, topic.Partition)
 
+	topic, err = ParseTopicName("my-tenant/my-cluster/my-namespace/my-topic")
```

Comment:
nitpick move this to the bottom where the other V1 topic name tests are.

Reply:
fixed